### PR TITLE
id-utils biome

### PR DESCRIPTION
**Pull request by CarlosGamero**
## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/background-jobs-common/vitest.config.mts [hunk 1]
```diff
@@ -5,7 +5,6 @@ import defineConfig from '@lokalise/package-vite-config/package'
 // @ts-ignore
 import packageJson from './package.json'
 
-/* eslint-disable import/no-default-export */
```
Comment (kibertoad) [inline packages/app/background-jobs-common/vitest.config.mts:8]: we don't need a biome override here?

File: packages/app/id-utils/package.json [hunk 2]
```diff
@@ -1,53 +1,48 @@
...
+    "ulid-uuid-converter": "^1.0.4",
+    "ulidx": "^2.3.0",
+    "uuidv7": "^1.0.1"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "^1.8.3",
+    "@lokalise/biome-config": "^1.3.0",
+    "@lokalise/package-vite-config": "latest",
+    "@vitest/coverage-v8": "^2.0.4",
+    "rimraf": "^5.0.7",
```
Comment (kibertoad) [inline packages/app/id-utils/package.json:43]: let's bump this to 6.x now that we are at it

File: packages/app/id-utils/package.json [hunk 3]
```diff
@@ -1,53 +1,47 @@
...
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    }
+  },
+  "scripts": {
+    "build": "rimraf dist && tsc",
```
Comment (kibertoad) [inline packages/app/id-utils/package.json:23]: why remove vite builds? wouldn't that break compatibility with browser and ESM?

Comment (CarlosGamero) [reply packages/app/id-utils/package.json:23]: Mmm good point, will put it back, sorry

File: packages/app/id-utils/package.json (hunk 2 above)
Comment (CarlosGamero) [reply packages/app/id-utils/package.json]: Done, thank youuu 🙏

**Review by kibertoad — APPROVED** [review]

File: packages/app/background-jobs-common/vitest.config.mts (hunk 1 above)
Comment (CarlosGamero) [reply packages/app/background-jobs-common/vitest.config.mts:8]: Not ask me why, but is not needed on `vitest.config` override, maybe it is disabled by default on biome

**Pull request merged**